### ISSUE TITLE
[COGNITION]: GIBCT EYB - Firefox MUST show fields in error correctly

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10168]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

While testing the estimate your benefits flow, I noticed Firefox alone and Firefox with NVDA were telling me a lot of fields were in error. After some inspection, I realized it had to do with the `pattern` attribute on our dollar amount fields. We have a **$** that is causing the browser's default error field behavior and styling to kick in. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Firefox does not show fields in error unless the pattern actually is broken
- [ ] NVDA does not announce the field as having invalid entry unless I have actually entered a bad value

## Environment

* Windows 10
* Firefox
* NVDA
* https://staging.va.gov/gi-bill-comparison-tool/profile/11806124
* https://staging.va.gov/gi-bill-comparison-tool/profile/11806124

## WCAG or Vendor Guidance (optional)

* [Error Prevention (Legal, Financial, Data): Understanding SC 3.3.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-reversible.html)

## Screenshots or Trace Logs

![Screenshot (38)](https://user-images.githubusercontent.com/934879/84528328-1a5b6500-aca5-11ea-92f0-f40fcb915c71.png)

**Comment by ksantiagomorgan6** [comment]
pull request https://github.com/department-of-veterans-affairs/vets-website/pull/13122

**Comment by sonntag-adam** [comment]
NVDA Firefox no longer showing fields as an error.

![Screen Shot 2020-06-15 at 12.13.18 PM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/3e78b18d-a0bd-481f-b823-3f74d889488f)
##
![Screen Shot 2020-06-15 at 12.13.39 PM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/a99484bd-d71e-45a8-bc9d-d02f418cdff8)

**Comment by sonntag-adam** [comment]
@1Copenut this is in staging ready for your review

**Comment by 1Copenut** [comment]
@sonntag-adam Verified on Firefox MacOS and Win10 on Staging. Moving to close.
